index.html

```diff
@@ -8,11 +8,100 @@
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <script>
+        function toggleMenu() {
+            var navBar=document.getElementById("navigation");
+            if (navBar.classList.contains("nav-mobile")) {
+                navBar.classList.remove("nav-mobile");
+                navBar.classList.add("navigation");
+            } else {
+                navBar.classList.remove("navigation");
+                navBar.classList.add("nav-mobile");
+            }
+            window.addEventListener("resize", function(event) {
+            if (navBar.classList.contains("nav-mobile")) {
+                navBar.classList.remove("nav-mobile");
+                navBar.classList.add("navigation");
+            }
+        })
+        }
+        
+    </script>
 </head>
 <body>
 
+    <header> 
+        <div class="logo">
+            <img src="./img/karma-logo.svg" alt="Karma Logo" />
+        </div>        
+        <nav>
+            <ul class="navigation" id="navigation">
+                <li><a href=""><strong>Meet Karma</strong></a></li>
+                <li><a href="">How it Works</a></li>
+                <li><a href="">Store</a></li>
+                <li><a href="">Blog</a></li>
+                <li><a href="">Help</a></li>
+                <li><a href="">Login</a></li>
+            </ul>
+        </nav>
+        <div class="hamburger">
+            <a href="#" onclick="toggleMenu()">
+                <img src="./img/menu-hamburger.svg" alt="Show Navigation"/>
+            </a>
+        </div>
+    </header>
+    <main>
+        <div class="hero">
```

Comment:
```suggestion
        <header class="hero">
```

Looking strong on the code side. By the way, you can have multiple headers on a page. You can have a header for a section, article, main, etc. Here, you could try associating a header with the main region

https://www.w3.org/WAI/tutorials/page-structure/regions/